collectors/carbonblack/utils.js

```diff
@@ -0,0 +1,171 @@
+var request = require('request');
+
+const Audit_Log_Events = 'AuditLogEvents';
+const Search_Alerts = 'SearchAlerts';
+const Search_Alerts_CB_Analytics = 'SearchAlertsCBAnalytics';
+const Search_Alerts_Vmware = 'SearchAlertsVmware';
+const Search_Alerts_Watchlist = 'SearchAlertsWatchlist';
+
+function getAPILogs(apiDetails, accumulator, state, clientSecret, clientId, maxPagesPerInvocation) {
+    let nextPage;
+    if (apiDetails.method === "GET") {
+        return new Promise(function (resolve, reject) {
+            request({
```

Comment:
We have our own wrapper around the request library that is used in other collectors, it's in al-collector-js [here](https://github.com/alertlogic/al-collector-js/blob/master/al_util.js#L52). I suggest using that for maintainability and consistency reasons